packages/account/src/Configs/poi-error-config.tsx

```diff
@@ -0,0 +1,13 @@
+import { Localize } from '@deriv/translations';
+import React from 'react';
+
+export const GENERIC_ERROR_MESSAGE = (
```

Comment:
just a question: we are assigning this error message to data.error?.code. 
 will this string gets localized without refresh on changing language?